README.md

```diff
@@ -56,16 +56,25 @@ We have tested Ray on Golem on Ubuntu and on WSL. It should work on MacOS and sh
 This [quickstart](https://docs.golem.network/docs/creators/ray/quickstart) shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
 It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](https://docs.golem.network/docs/creators/ray/setup-tutorial)
 
+It is recommended to create a new directory and a clean virtual environment to play with the following.
+It will let you avoid cluttering your system's Python installation with unnecessary packages.
 
 ## Install software
 
-The first step is installing Ray on Golem (recommended within a clean venv). It will install Ray as a dependency.
+The first step is installing Ray on Golem. It will install Ray as a dependency.
 
 ```bash
 # install ray-on-golem and ray (recommended within a clean venv)
 pip3 install -U ray-on-golem
 ```
 
+For now, you also need to download and install Golem node software representing you in the Golem network.
+
+```bash
+# install yagna - golem network daemon
+curl -sSf https://join.golem.network/as-requestor | bash -
```

Comment:
golem-node is added as dependency to `ray-on-golem` in this PR https://github.com/golemfactory/ray-on-golem/pull/93

Reply:
yes, I will be working on this change across all docs today.
this PR is keeping up with the previous changes that landed on the docs...